crates/database/src/orders.rs

```diff
@@ -132,22 +132,22 @@ INSERT INTO orders (
 VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20)
     "#;
     sqlx::query(QUERY)
-        .bind(order.uid)
-        .bind(order.owner)
+        .bind(&order.uid)
+        .bind(&order.owner)
```

Comment:
These changes aren't required but it felt more consistent to pass large types by reference.